src/lib.rs

```diff
@@ -1280,6 +1280,73 @@ impl VM {
         }
     }
 
+    pub fn set_slot_new_foreign_scratch<M: AsRef<str>, C: AsRef<str>, T: 'static + ClassObject>(
+        &self,
+        module: M,
+        class: C,
+        object: T,
+        slot: SlotId,
+        scratch: SlotId,
+    ) -> Result<&mut T, ForeignSendError> {
+        self.ensure_slots(slot + 1);
+        let conf = unsafe { &mut *(wren_sys::wrenGetUserData(self.vm) as *mut UserData) };
+
+        self.ensure_slots((slot + 1) as usize);
```

Comment:
I'm not sure if we need the `usize` cast here. (I might have just written it that way, and if I did, you can just resolve this comment.)

Reply:
Yes it was already there